hazelcast/src/main/java/com/hazelcast/spi/properties/ClusterProperty.java

```diff
@@ -1029,8 +1029,6 @@ private int getWhenNoSSLDetected() {
     public static final HazelcastProperty CLIENT_PROTOCOL_UNVERIFIED_MESSAGE_BYTES =
             new HazelcastProperty("hazelcast.client.protocol.max.message.bytes", 4096);
 
-    public static final HazelcastProperty AUDIT_LOG_ENABLED = new HazelcastProperty("hazelcast.auditlog.enabled", false);
```

Comment:
Can't we enable it by default? And at least dump everything into diagnostics. Diagnostics should also be enabled by default but that is another discussion.